api_spec.yaml

```diff
@@ -1,7 +1,11 @@
 openapi: 3.0.0
 info:
-  description: This is the API exposed by the SuperTokens Core. To be consumed by your backend only.
-  version: 2.16.2
+  description: |
```

Comment:
try using `<tenantId>` instead of `{tenantId}` in the paths

Reply:
Done